cmd/scrape/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/google/go-github/github"
+	fixtures "github.com/src-d/lookout-test-fixtures"
+)
+
+func main() {
+	tr := github.BasicAuthTransport{
+		Username: os.Getenv("GITHUB_USER"),
+		Password: os.Getenv("GITHUB_TOKEN"),
+	}
+
+	client := github.NewClient(tr.Client())
+
+	fmt.Println("Saving JSON fixtures...\n")
+
+	fixtureList := fixtures.GetAll()
+
+	for _, fixture := range fixtureList {
+		pr, _, err := client.PullRequests.Get(context.Background(),
+			fixture.URL.Owner, fixture.URL.Repo, fixture.URL.Number)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		data, err := json.MarshalIndent(pr, "", "  ")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = ioutil.WriteFile(
+			fixture.FilenamePR(fixture.CurrentRevision),
+			data, 0644)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		cc, _, err := client.Repositories.CompareCommits(context.Background(),
+			fixture.URL.Owner, fixture.URL.Repo,
+			pr.GetBase().GetSHA(), pr.GetHead().GetSHA())
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		data, err = json.MarshalIndent(cc, "", "  ")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = ioutil.WriteFile(
```

Comment:
do we really always want to rewrite existing fixture? State at github can evolve but we need some specific snapshot, no?